python/nav/etc/webfront/webfront.conf

```diff
@@ -136,3 +136,10 @@ enabled = no
 # security features that are only available with TLS/SSL enabled. In
 # development mode this defaults to `no`.
 # needs_tls = no
+
+#
+# Configuration items for QR codes in NAV
+#
+[qr_codes]
+# Whether to produce QR codes as SVG or PNG files
+file_format = png
```

Comment:
What happens if the user puts `foobar`, or even `PNG` (upper case) as the value here? Or any invalid value, for that matter?